notebooks/Composer_Functional.ipynb

```diff
@@ -0,0 +1,306 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = Net()\n",
+    "# only use one data augmentation since our small model runs quickly\n",
+    "# and allows the dataloader little time to do anything fancy\n",
+    "trainloader, testloader = make_augmented_dataloaders(['colout'])\n",
+    "train_and_eval(model, trainloader, testloader)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As we might expect, adding data augmentation doesn't help us when we aren't training long enough to start overfitting.\n",
+    "\n",
+    "Now let's try using some algorithms that modify the model. We're going to keep things simple and just add a [Squeeze-and-Excitation](https://docs.mosaicml.com/en/latest/method_cards/squeeze_excite.html) module after the larger of the two conv2d operations in our model. We show examples of how to use other algorithms as well as a reference."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def modify_model(model: torch.nn.Module, which_algorithms: Sequence[str]):\n",
+    "    if 'blurpool' in which_algorithms:\n",
+    "        cf.apply_blurpool(model)\n",
+    "    if 'squeeze_excite' in which_algorithms:\n",
+    "        # squeeze-excite can add a lot of overhead for small\n",
+    "        # conv2d operations, so only add it after convs with a\n",
+    "        # minimum number of channels\n",
+    "        cf.apply_squeeze_excite(model, latent_channels=64, min_channels=16)\n",
+    "    if 'ghost_batchnorm' in which_algorithms:\n",
+    "        cf.apply_ghost_batchnorm(model, ghost_batch_size=16)\n",
+    "\n",
+    "modify_model(model, which_algorithms=['squeeze_excite'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's see how our model does with all of the above algorithms applied."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_and_eval(model, trainloader, testloader)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Looks like adding squeeze-excite gives us another few percentage points of accuracy, and does so with little decrease in the iterations per second. Great!\n",
+    "\n",
+    "Of course, this is a toy model and an unrealistically short training duration, so almost nothing should be read into these results. But hopefully this served as a useful illustration of how to use Composer's algorithms without the Composer trainer, and with minimal change to your code. If you hit any problems or have questions, feel free to [open an issue](https://github.com/mosaicml/composer/issues/new/) or reach out to us [on Slack](https://join.slack.com/t/mosaicml-community/shared_invite/zt-w0tiddn9-WGTlRpfjcO9J5jyrMub1dg)."
```

Comment:
suggest removing this qualifer, i think its understood!

also can you link to the method cards overview? each one will have code to show how to use it the functional form.

Reply:
removed the qualifier.

Tried to link to the methods overview (`method_cards/methods_overview.html`) but it isn't live on the website, and we don't support `/en/latest/`, so I couldn't find a way to do it that didn't hardcode the library version. Happy to add post-release, or if you know a reliable url to use.